week14-18-05-03.md

```diff
@@ -0,0 +1,250 @@
+
+### Fix a memory leak in QueueHandler
+[@MrMage](https://github.com/MrMage)
+
+[vapor/core#134](https://github.com/vapor/core/pull/134)
+
+### Follow API change in Crypto
+[@t-ae](https://github.com/t-ae)
+
+[vapor/core#167](https://github.com/vapor/mysql/pull/167)
+
+### Add extra HTTPHeaders to WebSocket client
+[@pvels](https://github.com/pvels)
+
+[vapor/websocket#2](https://github.com/vapor/websocket/pull/2)
+
+### Adds git hash capability to vapor cloud
+Adds ```vapor cloud git-hash```
+
+[@joscdk](https://github.com/joscdk)
+
+[vapor/toolbox#221](https://github.com/vapor/toolbox/pull/221)
+
+### Adding Host header to requests
+[@givip](https://github.com/givip)
+
+[vapor/http#269](https://github.com/vapor/http/pull/269)
+
+### Fixed FormURLEncoded Decoder Media Type Check
+[@calebkleveter](https://github.com/calebkleveter)
+
+[vapor/http#266](https://github.com/vapor/http/pull/266)
+
+### Adds TLS to HTTPClient
+[@vkill](https://github.com/vkill)
+
+[vapor/http#260](https://github.com/vapor/http/pull/260)
+
+### Fix Timestampable when the models are structs
+[ccrazy88](https://github.com/ccrazy88)
+
+[vapor/fluent#445](https://github.com/vapor/fluent/pull/445)
+
+### Fix MySQL transaction support
+[@jseibert](https://github.com/jseibert)
+
+[vapor/fluent-mysql#100](https://github.com/vapor/fluent-mysql/pull/100)
+
+### Release pooled conn when catch for requestPooledConnection
+[@vkill](https://github.com/vkill)
+
+[vapor/database-kit#25](https://github.com/vapor/database-kit/pull/25)
+
+### Add url config to redis
+[@pedantix](https://github.com/pedantix)
+
+[vapor/redis#99](https://github.com/vapor/redis/pull/99)
+
+### Fix synchronization issue in QueueHandler
+[@jseibert](https://github.com/jseibert)
+
+[vapor/core#118](https://github.com/vapor/core/pull/118)
+
+### Fix infinite recursion in MySQLCharacterSet.description
+[@jseibert](https://github.com/jseibert)
+
+[vapor/mysql#155](https://github.com/vapor/mysql/pull/155)
+
+### Fixed issue with EvenLoop assert
+When sending asynchronously a ByteBuffer in a WebSocket, it fails since the current execution is not in the same EventLoop as the WebSocket handler.
+
+[@kikettas](https://github.com/kikettas)
+
+[apor/http#254](https://github.com/vapor/http/pull/254)
+
+### Fix Cloud.yml
+Fix the ```cloud.yml``` file so it uses the latest commands required to run an app
+
+[@0xTim](https://github.com/0xTim)
+
+[vapor/api-template#55](https://github.com/vapor/api-template/pull/55)
+
+### Export Async to avoid an import when conforming to Fluent protocols
+[@0xTim](https://github.com/0xTim)
+
+[vapor/fluent#417](https://github.com/vapor/fluent/pull/417)
+
+### Default port in MySQLDatabaseConfig to 3306
+[@0xTim](https://github.com/0xTim)
+
+[vapor/mysql#146](https://github.com/vapor/mysql/pull/146)
+
+### Fix MySQL index syntax
+[@jseibert](https://github.com/jseibert)
+
+[vapor/fluent-mysql#88](https://github.com/vapor/fluent-mysql/pull/88)
+
+
+## Tagged releases
```

Comment:
We don't need to do it in this release if you don't have the time, but in future releases, we should link each of these to their tagging on GitHub